internal/keycloakb/configdbmodule.go

```diff
@@ -68,6 +119,58 @@ func (c *configurationDBModule) GetConfiguration(ctx context.Context, realmID st
 	return config, err
 }
 
+func (c *configurationDBModule) GetBackOfficeConfiguration(ctx context.Context, realmID string, groupNames []string) (dto.BackOfficeConfiguration, error) {
+	var res = make(dto.BackOfficeConfiguration)
+	for _, slice := range splitIntoSlices(groupNames, 5) {
```

Comment:
Maybe we could do something like that: https://stackoverflow.com/questions/20271123/how-to-execute-an-in-lookup-in-sql-using-golang ?